### DISCUSSION TITLE
What Exactly Does the Hidden Device in BlackHole 4.0 Do?

### DISCUSSION THREAD

**Discussion by deddydrip** [ExistentialAudio/BlackHole#547] [General]
Just curious. Can I still use it with applications such as Logic or Loopback? If so, how do I select it, if it is hidden?

My first thought would be: it is an audio device that can be used by applications, but won't show up in any output selections.

Thank you for your hard work!

**Comment by devinroth** [comment]
It was a request from some developers. Yes you basically described exactly what it is, but it can only be used by applications that know the UID. Essentially it allows a custom app to capture the output without the users seeing the loopback input.

If you unhide the device that you have two identical clones of the same driver.

Hope that makes sense.

Devin
